<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWindow</class>
 <widget class="QMainWindow" name="StartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StartWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>276</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="loginLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="usernameLayout">
       <item>
        <widget class="QLabel" name="userNameLabel">
         <property name="text">
          <string>username:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="usernameLineEdit"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="passwdLayout">
       <item>
        <widget class="QLabel" name="passwdLabel">
         <property name="text">
          <string>password:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="passwdLineEdit"/>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="signInBtn">
         <property name="text">
          <string>SignIn</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="signUpBtn">
         <property name="text">
          <string>SignUp</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
